packages/devtools_app/lib/src/config_specific/flutter/import_export/_import_export_base.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../../flutter/controllers.dart';
+import '../../../flutter/notifications.dart';
+import '../../../timeline/timeline_controller.dart';
+
+abstract class ImportControllerBase<T> {
```

Comment:
using imports rather than exports what you can do is change this class to not have the word Base
and provide a factory constructor that uses the conditional imports to generate an instance of the correct version depending on the context.